src/frontend/Komponenter/Behandling/Totrinnskontroll/OppgaverForOpprettelse.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import { useEffect } from 'react';
+import { Checkbox } from '@navikt/ds-react';
+import { Behandlingstype } from '../../../App/typer/behandlingstype';
+import { Behandling } from '../../../App/typer/fagsak';
+import { OppgaverForOpprettelseState } from '../../../App/hooks/useHentOppgaverForOpprettelse';
+import { AlertError } from '../../../Felles/Visningskomponenter/Alerts';
+import { FamilieCheckboxGroup } from '@navikt/familie-form-elements';
+import styled from 'styled-components';
+import { AGrayalpha300 } from '@navikt/ds-tokens/dist/tokens';
+export const HvitTekst = styled.div`
+    color: white;
+`;
+
+const CheckboxGruppe = styled(FamilieCheckboxGroup)`
+    width: 30rem;
+    margin: 0.5rem 1rem 0.5rem 1rem;
+    padding: 0 0.25rem 0 0.25rem;
+    background-color: ${AGrayalpha300};
+    border-radius: 5px;
+`;
+
+export enum OppgaveForOpprettelseType {
```

Comment:
Navnendring: OppgaveTypeForOpprettelse for å vise tydeligere at dette er et enum med oppgavetyper og ikke en type i seg selv